approval_polls/templates/approval_polls/my_polls.html

```diff
@@ -10,16 +10,50 @@
     <div class='row-fluid top-buffer'>
       <div class='col-md-12'>
         {% for poll in latest_poll_list %}
-          <div class='well well-sm'>
+          <div class='well well-sm' id='poll{{ poll.id }}'>
             <a href="{% url 'approval_polls:detail' poll.id %}">{{ poll.question }}</a>
-            <a class='pull-right btn btn-danger btn-xs'> remove </a>
+            <a class='pull-right btn btn-danger btn-xs' onClick="showAlert({{ poll.id }})"> remove </a>
```

Comment:
It will be better to move all event handlers to the new JS file that you create.